proposals/2021/Delete Install or Upgrade Master Venting.md

```diff
@@ -0,0 +1,12 @@
+# Delete Install or upgrade master venting
+
+## Overview
+
+This proposal is to address the issue of the redundant `Install or upgrade master venting` measure option under `ChilledWaterHotWaterAndSteamDistributionSystems` and `Install or Upgrade Master Venting` measure option under `OtherHVAC`.
+
+## Justification
+
+`Install or upgrade master venting` measure repeats in `ChilledWaterHotWaterAndSteamDistributionSystems` and `OtherHVAC`, and we should keep only one of them. This proposal suggests that we remove it under `ChilledWaterHotWaterAndSteamDistributionSystems` and keep it under `OtherHVAC`, as master venting approach could be implemented not only in chilled or hot water system.
```

Comment:
@nllong could we justify whether we should keep it under `ChilledWaterHotWaterAndSteamDistributionSystems` or `OtherHVAC`? Any reference to look up?

Reply:
I would check ASHRAE Standard 100. 
![image](https://user-images.githubusercontent.com/1907354/116938188-96e19400-ac27-11eb-93b4-191209038457.png)